### ISSUE TITLE
installing via omf requires a fish reload to operate, works immediately after install with fisher

### ISSUE THREAD

**Issue by allixsenos** [laughedelic/pisces#25]
on a fresh Mac, I'm trying out fish... I have very few things installed yet pisces doesn't work. I have no clue how to debug. help? :)

**Comment by laughedelic** [comment]
Hi @allixsenos! if you want more interactive debugging help, let's move to the [Gitter chat](https://gitter.im/laughedelic/pisces). 

You can start by telling me how you installed it: are you using fisher or omf?

**Comment by allixsenos** [comment]
I figured out a proper repro case...

```
# OMF - pisces requires a reload of the environment (starting a new fish)
docker run -it ubuntu
apt-get update && apt-get install git curl fish
curl -L https://get.oh-my.fish | fish
omf install pisces
echo "| # doesn't add the other "
^D
fish
echo "|"  # works!

# fisher - pisces works immediately after fisher add
docker run -it ubuntu
apt-get update && apt-get install git curl fish
fish
curl https://git.io/fisher --create-dirs -sLo ~/.config/fish/functions/fisher.fish
fisher add laughedelic/pisces
echo "|"  # works!
```

**Comment by laughedelic** [comment]
@allixsenos do you want to update the installation instructions for omf in the readme?

**Comment by allixsenos** [comment]
sure!
Alexey Alekhin <notifications@github.com> wrote:
“@allixsenos do you want to update the installation instructions for omf in the readme?”

**Comment by laughedelic** [comment]
Fixed in e45e0869855d089ba1e628b6248434b2dfa709c4.
